### PR TITLE
doc: Fixed board names in the Zigbee samples

### PR DESCRIPTION
**Pull request by umapraseeda**
ref: NCSDK-3933 and
https://github.com/nrfconnect/sdk-nrf/pull/2425
Fixed board names in Zigbee samples

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: samples/zigbee/network_coordinator/README.rst [hunk 1]
```diff
@@ -16,17 +16,19 @@ It is a minimal implementation that supports only the network steering commissio
...
+The sample supports the following development kits:
 
-  * |nRF52840DK|
-  * |nRF52833DK|
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set8_start
+   :end-before: set8_end
 
-* One or both of the following samples:
+You can use one or more of the development kits listed above and mix different development kits.
```
Comment (ru-fu) [inline samples/zigbee/network_coordinator/README.rst:25]: I think here you use only one DK, not one or more?

Comment (maciekfabia) [reply samples/zigbee/network_coordinator/README.rst:25]: yes, there should be one network coordinator, not more

File: samples/zigbee/network_coordinator/README.rst [hunk 2]
```diff
@@ -16,17 +16,19 @@ It is a minimal implementation that supports only the network steering commissio
...
 
-  * The :ref:`Zigbee light bulb <zigbee_light_bulb_sample>` sample programmed on one or more separate devices.
-  * The :ref:`Zigbee light switch <zigbee_light_switch_sample>` sample programmed on one or more separate devices.
+For this sample to work, one or both of the following samples need to be programmed:
```
Comment (maciekfabia) [inline samples/zigbee/network_coordinator/README.rst:27]: The following samples (light bulb, light switch) are not mandatory. They are optional,  **Testing** section explains that these samples are optional and testing steps that involve them can be skipped

File: samples/zigbee/network_coordinator/README.rst (hunk 1 above)
Comment (umapraseeda) [reply samples/zigbee/network_coordinator/README.rst:25]: Changed.

File: samples/zigbee/light_switch/README.rst [hunk 3]
```diff
@@ -25,15 +25,18 @@ The sleepy behavior can be enabled by pressing **Button 3** while the light swit
 Requirements
 ************
 
-* One or more of the following development kits:
+The sample supports the following development kits:
```
Comment (greg-fer) [inline samples/zigbee/light_switch/README.rst:28]: @ru-fu , @umapraseeda , should this change and changes in lines 33-39 be applied to all samples in NCS?

Comment (ru-fu) [reply samples/zigbee/light_switch/README.rst:28]: Yes, that's the plan - see https://github.com/nrfconnect/sdk-nrf/pull/2425
But it's impossible to get in all changes at the same time, that's why it's split up in separate PRs. :)

**Pull request merged**